checkmarx/util.py

```diff
@@ -306,6 +303,9 @@ def scan_gh_artifact(
 
 
 def greatest_severity(result: model.ScanResult) -> model.Severity | None:
+    if not result.scan_statistic:
```

Comment:
raise (e.g. runtime-error); explicitly check for error-state? if there is not scan-statistics, result must be erronrous (scan failed). in that case, it is an error to read severity.